packages/harpguru-core/src/components/harp-face/hooks/use-styles.ts

```diff
@@ -13,6 +13,7 @@ export const useStyles = (): HarpFaceStyles => {
   const { dynamicSizes } = useSizes()
   const { columnWidth, rowHeight, fragmentGutter } = dynamicSizes
 
+  // TODO: Replace with global state prop
   const { harpFaceRowCount, harpFaceColumnCount } = useLayoutFacts()
 
   const octaveColumnGroups = useOctaveColumnGroups()
```

Comment:
Should perhaps have a look at whether this function needs updating now too.

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/120#issuecomment-938452028